addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1552,6 +1556,15 @@ var SnippetsMenu = Widget.extend({
             }
         });
     },
+    /**
+     * @private
+     */
+    _activateTouchEvents() {
+        document.addEventListener("touchstart", this._touchHandler, true);
+        document.addEventListener("touchmove", this._touchHandler, true);
+        document.addEventListener("touchend", this._touchHandler, true);
+        document.addEventListener("touchcancel", this._touchHandler, true);
+    },
```

Comment:
As it is called only once, I would not make functions out of this. Just do that in the start and destroy and add a comment on top of it.